dashboard/pkg/epinio/store/epinio-store/actions.ts

```diff
@@ -228,17 +230,30 @@ export default {
     const spoofedSchemas = rootGetters['type-map/spoofedSchemas'](EPINIO_PRODUCT_NAME);
     const excludeInstances = spoofedSchemas.filter((schema: any) => schema.id !== EPINIO_TYPES.INSTANCE);
 
-    res.data = res.data.concat(excludeInstances);
+    const data = [
+      ...res.data,
+      ...excludeInstances,
+    ];
+
+    try {
+      const nodeMetricsSchema = await dispatch(`cluster/request`, { url: `/k8s/clusters/${ clusterId }/v1/schemas/${ METRIC.NODE }` }, { root: true });
```

Comment:
Sorry, missed this from the previous review. We should only do this if we're in the embedded world, it will not work in standalone so will always fail.

Reply:
hmm, gh fail. Line of comment is out. should be 239

Reply:
Only the line was wrong on this one. This should still be gated on not a single product